src/seth/default.nix

```diff
@@ -6,10 +6,11 @@ stdenv.mkDerivation rec {
   version = "0.9.0";
   src = ./.;
 
+  buildInputs = [ nodejs ];
```

Comment:
why is this needed? `nodejs` is already added to path in `postinstall`?

Reply:
This is needed so that `patchShebangs` finds `noejs` in the `PATH` and therefore patches it.

I'll see if it's possible to remove the repetition, but it could be weird if only `nodejs` is treated differently compared to all the other packages in the `path` variable below.

Reply:
But the reason why patching the `#!/usr/bin/env node` shebang is necessary for seth used in `dapp-tests` (and isn't necessary for seth used from the shell to work) is not clear to me.